### PR TITLE
Kernel.binary_part/3 doc edits

### PR DESCRIPTION
**Pull request by bradhanks**
The wrong variable name was being used.

### CODE REVIEW

**Comment by whatyouhide** [comment]
@bradhanks thanks for the contribution! When you added `iex>`, it now treats that as a doctest, so you'll need to either fix it so that it passes or remove the `iex>` from the example 🙃

**Comment by bradhanks** [comment]
> @bradhanks thanks for the contribution! When you added `iex>`, it now treats that as a doctest, so you'll need to either fix it so that it passes or remove the `iex>` from the example 🙃

I didn't mean to add that. I'll remove it or fix it :)

File: lib/elixir/pages/getting-started/writing-documentation.md
Comment (whatyouhide) [inline lib/elixir/pages/getting-started/writing-documentation.md:1]: These are a lot of unrelated changes. Can you remove them?

Comment (bradhanks) [reply lib/elixir/pages/getting-started/writing-documentation.md:1]: yeah sorry I had a brief technical issue.

**Comment by whatyouhide** [comment]
Thanks @bradhanks 💟

**Comment by sabiwara** [comment]
I wonder if it wouldn't be better to rename the arg to `size` to align with its counterpart [`binary_slice/3`](https://hexdocs.pm/elixir/Kernel.html#binary_slice/3), and would also be more consistent regarding this [convention](https://hexdocs.pm/elixir/main/naming-conventions.html#length-and-size)?

**Comment by josevalim** [comment]
Agreed!
